qt/scientific_interfaces/Muon/test/ALCDataLoadingPresenterTest.h

```diff
@@ -131,10 +148,15 @@ class ALCDataLoadingPresenterTest : public CxxTest::TestSuite {
   }
 
   void setUp() override {
-    m_view = new NiceMock<MockALCDataLoadingView>();
-    m_presenter = new ALCDataLoadingPresenter(m_view);
+    m_view = std::make_unique<NiceMock<MockALCDataLoadingView>>();
+    auto model = std::make_unique<ALCDataLoadingModel>();
+    m_presenter = std::make_unique<ALCDataLoadingPresenter>(m_view.get(), std::move(model));
```

Comment:
It would be useful to keep a member variable which is a non-owning raw pointer to the model. This means that we will be able to assert expectations on the model in the below unit tests by using the `m_model` member variable. (See the 3rd green line below)
```suggestion
    m_view = std::make_unique<NiceMock<MockALCDataLoadingView>>();
    auto model = std::make_unique<ALCDataLoadingModel>();
    m_model = model.get();
    m_presenter = std::make_unique<ALCDataLoadingPresenter>(m_view.get(), std::move(model));
```

Reply:
I see!